src/details/ArborX_DetailsUtils.hpp

```diff
@@ -80,12 +80,13 @@ void exclusivePrefixSum(Kokkos::View<ST, SP...> const &src,
   using ExecutionSpace =
       typename Kokkos::ViewTraits<DT, DP...>::execution_space;
   using ValueType = typename Kokkos::ViewTraits<DT, DP...>::value_type;
+  using MemorySpace = typename Kokkos::ViewTraits<DT, DP...>::memory_space;
 
   auto const n = src.extent(0);
   ARBORX_ASSERT(n == dst.extent(0));
   Kokkos::parallel_scan(
       "exclusive_scan", Kokkos::RangePolicy<ExecutionSpace>(0, n),
-      Details::ExclusiveScanFunctor<ValueType, ExecutionSpace>(src, dst));
+      Details::ExclusiveScanFunctor<ValueType, MemorySpace>(src, dst));
```

Comment:
```suggestion
      Details::ExclusiveScanFunctor<ValueType, DeviceType>(src, dst));
```

Reply:
Except that `DeviceType` is also not declared. Do you feel strongly about using `DeviceType` or `MemorySpace` as second template argument? For me, it just looks more appropriate to use a memory space even if you could also pass a device type.

Reply:
You're right sorry.  I prefer to stick with the device type for consistency with the rest unless you have a good reason not to.

Reply:
I don't feel like fighting for it. I changed to `DeviceType`.